### PR TITLE
CFR header style updates

### PR DESCRIPTION
**Pull request by xtine**
* Style intro CFR change description.
* Correct section header width per eregs/notice-and-comment#130

NOTE: There's a slight issue with CFR ToC links where it appends the hash of the CFR section number so it will anchor page load to the section number instead to the top of the page that shows the change description.

### CODE REVIEW

**Comment by cmc333333** [comment]
Heads up, branch conflict

File: regulations/templates/regulations/preamble-partial.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {% load macros render_nested %}
 
 <section id="{{full_id}}" class="reg-section" data-page-type="preamble-section" data-doc-id="{{doc_number}}">
-  <ul id="preamble-read">
+  <div id="preamble-read">
     {% render_nested template=sub_template context=sub_context %}
-  </ul>
+  </div>
```
Comment (cmc333333) [inline regulations/templates/regulations/preamble-partial.html]: I think we'll need to nest an `ol` inside this div -- don't the subtemplates generate `li`s?

File: regulations/templates/regulations/cfr_changes.html [hunk 2]
```diff
@@ -1,9 +1,17 @@
 {% for instruction in instructions %}
-  <div style="border: 1px solid black;">{{ instruction }}</div>
+  <div class="cfr-changes">
```
Comment (cmc333333) [inline regulations/templates/regulations/cfr_changes.html]: :D

Comment (cmc333333) [inline regulations/templates/regulations/cfr_changes.html]: Quick note here: using a term like "instructions" might be more helpful -- "change" takes on a lot of different meanings in our code.

File: regulations/templates/regulations/preamble-partial.html (hunk 1 above)
Comment (xtine) [inline regulations/templates/regulations/preamble-partial.html]: When I changed this I was looking at CFR sub_context and saw it was returning `<section>` and `<ol>`s in them, not realizing the regular preamble was returning the list items... I'll change it back and instead wrap CFR subtext in a list item instead. 😑

**Comment by cmc333333** [comment]
We'll figure out the templates; I'm betting they've not been right from the beginning.

**Pull request merged**